tests/pkg_cayenne-lpp/main.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     test
+ * @{
+ *
+ * @file
+ * @brief       Cayenne Low Power Payload example application
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "cayenne-lpp.h"
+
+static cayenne_lpp_t lpp;
+
+int main(void)
+{
+    puts("Cayenne LPP test application");
+
+    cayenne_lpp_add_temperature(&lpp, 3, 27.2);
```

Comment:
Just a minor thing... maybe add labels to the numeric values? Just to make them easier to read and understand params

Reply:
> maybe add labels to the numeric values?

I don't understand. Do you mean adding a comment beside the values ?

Reply:
I meant via #define.

Reply:
Ok, I'll add a comment because these values doesn't come from nowhere, these are the values given as example in the cayenne lpp documentation. Then it's easy to verify that the generated payload is correct.

Reply:
ok, perfect

Reply:
I applied the same values given as example, but they don't give the exact same result. I don't know where the issue come from (cayenne-lpp library or their doc).

Reply:
@aabadie so, if it's working in TTN, do you think this could be an issue with their documentation?

Reply:
> do you think this could be an issue with their documentation?

Could be. The first buffer (2 temperature sensors) works, the 2 others differ slightly. The C library is very simple, so I doubt there's an issue there.